client/disputes/details/index.tsx

```diff
@@ -27,11 +27,19 @@ const DisputeDetails = ( {
 	query: { id: disputeId },
 }: {
 	query: { id: string };
-} ): JSX.Element => {
+} ): JSX.Element | null => {
```

Comment:
I'm (now) in the habit of declaring React components as `Element | null`, consistent with classic React. As I understand, returning null is totally fine way to not render anything. 

Question for front end experts – is this a good approach? Should we have a standard type for this? What does the TypeScript React community do?

Reply:
Since as of currently, there is a possibility this component will return null, this `| null` addition is necessary. However, I argue that this component should not return null. See my comment below.

Reply:
Note that when we use `React.FC`/`React.FunctionComponent`, the return type is implicit and we don't have to define it (see [TS guidelines doc](https://github.com/Automattic/woocommerce-payments/blob/develop/docs/typescript/react-components.md#declare-the-component-using-the-attribute-type-you-declared-in-your-interface)).

I've switched this to `React.FC` in 2c4dbfc.

Reply:
Aha! Now I'm a fan of `React.FC` and will use from here on in. Can we encourage that pattern with a linter or tooling?

Reply:
Looking at the doc, we might be able to distil this to a principle:

- `Use React.FC type for React components`